src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/LeaseAlreadyPresentException.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
+
+public class LeaseAlreadyPresentException extends RuntimeException {
+    private static final long serialVersionUID = 6136150918143302940L;
+
+    public LeaseAlreadyPresentException(String message) {
```

Comment:
worth adding a cause if we're wrapping some existing exception, we'll know where it was thrown originally